src/screens/ENSAssignRecordsSheet.js

```diff
@@ -28,7 +35,11 @@ export default function ENSAssignRecordsSheet() {
         </Stack>
       </Box>
       <SheetActionButtonRow>
+        <TintButton onPress={handlePressBack} color="secondary60">
+          􀆉 Back
+        </TintButton>
         <SheetActionButton
+          color={'#9875D7'}
```

Comment:
I reckon we could just make this button the accent color (blue), in the designs, the accent color depends on the ENS avatar dominant color, but since we aren't doing that in phase 1, I think just having the app accent color will be good.
```suggestion
```